x-pack/plugins/security_solution/public/management/cypress/e2e/response_actions/response_console/file_operations.cy.ts

```diff
@@ -21,8 +21,7 @@ import { enableAllPolicyProtections } from '../../../tasks/endpoint_policy';
 import { createEndpointHost } from '../../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../../tasks/delete_all_endpoint_data';
 
-// Failing: See https://github.com/elastic/kibana/issues/172920
-describe.skip('Response console', { tags: ['@ess', '@serverless'] }, () => {
+kbdescribe('Response console', { tags: ['@ess', '@serverless'] }, () => {
```

Comment:
typo: `kbdescribe`

there were no valid test runs in the flaky runner: `No tests found - all tests could have been skipped via Cypress tags`

(sadly, there were no js/ts error either to make the test runner fail - i guess this is because of the ess/serverless test filtering logic, which eliminates the files before importing and processing them, so the unknown `kbdescribe` function was never called)

Reply:
oh, at least the `check types` step captured the error

Reply:
😅